Added languages to Huggle (GCI 2017 task)

Clockery opened the pull request:
Added a couple of languages which weren't called to core.cpp and text.qrc, as part of the Google Code-in task "Incorporate new languages into Huggle".

This is my first pull request, and I'm not sure whether I've done this right, so please do let me know if there's anything to be fixed :) thanks

benapetr commented on huggle/core.cpp:
```diff
@@ -619,6 +622,7 @@ void Core::LoadLocalizations()
     Localizations::HuggleLocalizations->LocalInit("pl");
     Localizations::HuggleLocalizations->LocalInit("pt"); // Portuguese
     Localizations::HuggleLocalizations->LocalInit("pt-BR"); // Portuguese (in Brazil)
+    Localizations::HuggleLocalizations->LocalInit("qqq");
```
qqq definitely shouldn't be here, that's not really a language ;)

benapetr commented:
Hi, please check if the language files you added really contains any useful data. Some of the .xml files we had didn't contain any localization. QQQ is documentation.

Clockery commented:
Sorry, my bad :) I've removed QQQ from the list.

Also, the km.xml file has just a couple of lines in it. The line calling it was commented out in core.cpp, and I removed the comment. Is that okay, or should I re-comment the line?

benapetr commented:
I would say that km.xml has only 1 line and that's the name of language, would only confuse users that could think it's real localization if they saw it.

Clockery commented:
I've commented it out again now, thanks :)
